src/kibana/styles/theme/_theme.less

```diff
@@ -9,24 +9,88 @@
 .navbar {
   border-width: 0;
 
+  box-shadow: inset 0 -10px 10px -12px #333;
+  -moz-box-shadow: inset 0 -10px 10px -12px #333;
+  -webkit-box-shadow: inset 0 -10px 10px -12px #333;
+
   &-default {
 
     .badge {
       background-color: #fff;
       color: @navbar-default-bg;
     }
+
+    .navbar-nav > .active > a {
+
+      @media (max-width: @screen-sm-min) {
+        &:before {
+          display: none !important;
+        }
+      }
+
+      &:before {
+        content:"";
+        display:inline-block;
+        position:absolute;
+        border:@navbar-arrow-size solid @body-bg;
+        border-color:transparent transparent @body-bg transparent;
+        top:(@navbar-height - @navbar-arrow-size*2);
+        left: 0;
+        right: 0;
+        margin: 0 auto;
+        width: 0;
+        height: 0;
+      }
+    }
+
   }
 
   &-inverse {
 
+    .logo {
+      background-image: url('../images/no_border.png');
+      height: @navbar-height;
+      width: @navbar-height;
+    }
+
     .badge {
       background-color: #fff;
-      color: @navbar-inverse-bg;
+      color: lighten(@navbar-inverse-bg, 10%);
     }
+
+    .navbar-nav > .active > a {
+
+      @media (max-width: @screen-sm-min) {
+        &:before {
+          display: none !important;
+        }
+      }
+
+      &:before {
+        content:"";
+        display:inline-block;
+        position:absolute;
+        border:@navbar-arrow-size solid @navbar-default-bg;
+        border-color:transparent transparent @navbar-default-bg transparent;
+        top:(@navbar-height - @navbar-arrow-size*2);
+        left: 0;
+        right: 0;
+        margin: 0 auto;
+        width: 0;
+        height: 0;
+      }
+    }
+
   }
 
   &-brand {
-    padding: 12px 15px 30px;
+
+    font-size: 1.8em;
+    background-color: lighten(@navbar-inverse-bg, 10%);
+
+    &:hover {
+      background-color: lighten(@navbar-inverse-bg, 10%) !important;
```

Comment:
@rashidkpc this really makes the branding feel clickable, even though it's not. Mind if I remove it?